docs/traffic-policy/actions/index.mdx

```diff
@@ -44,6 +44,16 @@ actions:
     categories:
       - security
 
+  - type: circuit-breaker
+    name: Circuit Breaker
+    description: Maintain system reliability by rejecting requests when error rates exceed defined thresholds.
+    phases:
+      - on_http_request
+      - on_http_response
```

Comment:
I think just `on_http_request` here

Reply:
Yep you're right—good catch & thank you!